content/docs/releases/release-notes/release-notes-1.16.md

```diff
@@ -14,6 +20,77 @@ so that platform teams can monitor the performance of all the cert-manager compo
+
+### ACME Issuer Route53 Solver
+
+The Route53 DNS01 solver code had become over-complicated due to its age and due
+to the variety of authentication methods that have been added over the years.
+When we upgraded to `AWS SDK for Go V2`in the last release, we did not have a
+good understanding of the new SDK and we were not able to test it thoroughly
+with all authentication methods. In this release we started putting that right.
+
+In this release we have tidied up the code and added more logging so that it is
+easier to debug problems in the field.
+We have improved the documentation of the Route53 API fields, particularly the region field,
+where we have tried to describe where and how cert-manager uses that value.
+
+We have relaxed the API validation so that the `region` field is now optional.
+cert-manager will now fall back to using the `AWS_REGION` environment variable of the controller Pod,
+regardless of which authentication mechanism is used.
+
+Users who use IAM Roles for Service accounts or Pod Identity need
+not specify the region, but if your Issuer or ClusterIssuer does include a region (for the sake of satisfying the old API validation),
+that issuer region will be ignored, if the `AWS_REGION` environment variable is set.
+
+Read the [ACME Issuer Route53](../../configuration/acme/dns01/route53.md) page to learn more.
+
+### Memory Optimizations
+
+We have continued our effort to reduce the memory footprint of cert-manager.
+
+The cainjector no longer caches Secret data; instead it only caches the metadata of Secret resources.
+This significantly reduces its memory usage.
+It also reduces the load on the Kubernetes API server, when cainjector starts up,
+because it no longer needs to send all the data of all the Secret resources over the network.
+
+We have added a new `ClientWatchList` feature flag to the controller, cainjector, and the webhook.
+This is actually a new beta feature in the Kubernetes client-go module,
+which enables a much more efficient mechanism for populating the client side caches.
+This reduces the load on the Kubernetes API server,
+because cert-manager components will no longer request complete unpaged lists of all API resources when they start up.
+And it reduces the peak memory use of the cert-manager components when they startup,
+because they no longer have to hold a duplicate unpaged list of resources in-memory
+while they add them to the client side cache.
+
+### Helm Schema Validation
+
+The Helm chart now includes a JSON schema which will validate the values that you supply when installing the chart.
+This will help you to get your Helm values right first time.
+It will alert you to typos and unrecognized fields in your existing Helm values files.
+And it will make it easier for the cert-manager maintainers to maintain the Helm chart,
+avoiding typos and mistakes in the default values file.
```

Comment:
suggestion: Not going to block this PR over this (you said it's WIP) but I think this should be at the top. I think the chances of this breaking a _lot_ of people are non-trivial and we need to warn people about it.

This is the thing in this release which is highest risk IMO!